tests/store/test_status_collaboration_filters.py

```diff
@@ -0,0 +1,34 @@
+from cg.store import Store
+from cg.store.models import Collaboration
+from cg.store.status_collaboration_filters import get_collaboration_by_internal_id
+
+
+def test_get_collaboration_by_internal_id(base_store: Store, collaboration_id: str):
+    """Test getting collaboration by internal_id."""
+    # GIVEN a store with a collaboration
+
+    # WHEN retrieving the collaboration
+    collaboration: Collaboration = get_collaboration_by_internal_id(
+        collaborations=base_store._get_collaboration_query(),
+        internal_id=collaboration_id,
+    ).first()
+
+    # THEN collaboration should be returned
+    assert collaboration
```

Comment:
We could replace these asserts with a more explicit one to ensure that the correct type is returned
```python
assert isinstance(collaboration, Collaboration)
```

Reply:
Good call 👍